cvm.mk

```diff
@@ -153,20 +153,24 @@ cvm8: jdk8vm17
 
 cvm8default17: jdk8vm17
 	echo "-server17 KNOWN" > $(CVM8_LIBDIR)/jvm.cfg
+	echo "-cvm KNOWN" >> $(CVM8_LIBDIR)/jvm.cfg
 	echo "-server KNOWN" >> $(CVM8_LIBDIR)/jvm.cfg
 	echo "-client IGNORE" >> $(CVM8_LIBDIR)/jvm.cfg
 	echo "-server17 KNOWN" > $(OUTPUTDIR)/$(DISTRO_NAME)/jre/lib/$(ARCH_DIR)/jvm.cfg
+	echo "-cvm KNOWN" >> $(OUTPUTDIR)/$(DISTRO_NAME)/jre/lib/$(ARCH_DIR)/jvm.cfg
```

Comment:
can we generate `$(CVM8_LIBDIR)/jvm.cfg` and copy to `$(OUTPUTDIR)/$(DISTRO_NAME)/jre/lib/$(ARCH_DIR)/`?